templates/al2023/runtime/gpu/initialize_nvidia_clock.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
this seems like something that would be better handled by nodeadm - for reference https://github.com/clowdhaus/eksnode/blob/main/eksnode/src/gpu.rs

Reply:
you could do it as a systemd unit (with bash script) - I'm just not that good with some gnarly bash-fu 😅 

also, not all cards will accept the max settings

Reply:
It's gonna take a lot more golang to accomplish the same thing as the bash script 😄 plus we don't need to block on it in nodeadm. A separate unit makes sense to me, downstream of `nvidia-persistenced`

Reply:
Right now this file and loading the kernel modules are in the same unit. I can put them in their own.

Reply:
Why I ended up doing is:
`nodeadm -> load-kmod -> nvidia-persistenced -> init-clocks`